index.html

```diff
@@ -502,21 +617,21 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             "HTML/webappapis.html#concept-relevant-global">relevant global
             object</a>.
             </li>
-            <li>Let <i>notify list</i> be a copy of <a data-cite=
+            <li>Let <var>notify list</var> be a copy of <a data-cite=
             "HTML/webappapis.html#concept-relevant-global">relevant global
             object</a>'s <a>list of registered performance observer
             objects</a>.
             </li>
-            <li>For each <a>PerformanceObserver</a> object <i>po</i> in
-            <i>notify list</i>, run these steps:
+            <li>For each <a>PerformanceObserver</a> object <var>po</var> in
+            <var>notify list</var>, run these steps:
               <ol>
-                <li>Let <i>entries</i> be a copy of <i>po</i>’s <a>observer
+                <li>Let <var>entries</var> be a copy of <var>po</var>’s <a>observer
                 buffer</a>.
                 </li>
-                <li>Empty <i>po</i>’s <a>observer buffer</a>.
+                <li>Empty <var>po</var>’s <a>observer buffer</a>.
                 </li>
-                <li>If <i>entries</i> is non-empty, call <i>po</i>’s callback
-                with <i>entries</i> as first argument and <i>po</i> as the
+                <li>If <var>entries</var> is non-empty, call <var>po</var>’s callback
```

Comment:
Paging Dr. @marcoscaceres for opinions on `<i>` vs. `<var>` here

Reply:
As a rule, avoid `<i>` for variables. Always use `<var>` instead of `<i>`, unless you really mean "italics" (which would be weird?). ReSpec treats `var`s as special, allow readers to click on them to see where they are used. Additionally, `<var>`s will soon know about their IDL or Infra data type, making them more powerful.

Reply:
Ok, so `var` seems correct here.